.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js:
+  - "node"
+  - "6"
+  - "5"
```

Comment:
The numbers represent the versions we're testing. I'll add this to the `readme` about it. [Source](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions). 👍